packages/sdk/src/services/PlatfformManager/openDeeplink.ts

```diff
@@ -38,14 +37,27 @@ export function openDeeplink(
 
     // It should only open after we can acknowledge that the rpc call that triggered the deeplink has been sent
     // TODO how can we know that the rpc call has been sent?
-    if (typeof window !== 'undefined') {
-      let win: Window | null;
-      if (state.useDeeplink) {
-        win = window.open(deeplink, '_blank');
-      } else {
-        win = window.open(universalLink, '_blank');
+    if (state.useDeeplink) {
+      if (typeof window !== 'undefined') {
+        // We don't need to open a deeplink in a new tab
+        // It avoid the browser to display a blank page
+        window.location.href = deeplink;
       }
-      setTimeout(() => win?.close?.(), LINK_OPEN_DELAY);
+    } else {
```

Comment:
we should probably check `typeof document !== 'undefined"`, since we can't know we'll always have access to a document/dom